scripts/lib/auth.js

```diff
@@ -64,23 +80,36 @@ function getApiUrl(endpoint) {
 /**
  * Remove the authentication cookie
  *
+ * @{@link  https://github.com/js-cookie/js-cookie}
+ *
  * @return {undefined}
  */
 function removeAuthCookie() {
-    var cookieValue = 'REMOVED';
-    var domain = '.mrn.org';
-    var path = '/';
-    var name = getOptions().authCookieName;
-    document.cookie = [
-        name,
-        '=',
-        cookieValue,
-        '; Path=',
-        path,
-        '; Domain=',
-        domain,
-        ';'
-    ].join('');
+    var authCookieName = getOptions().authCookieName;
+    var hostPieces = location.hostname.split('.');
+    var options = {
+        path: '/'
+    };
+
+    if (hostPieces.length > 2) {
+        options.domain = '.' + hostPieces.slice(hostPieces.length - 2).join('.');
```

Comment:
You can just do hostPieces.slice(-2)` to grab the last two elements of an array.

proof:
```
> ['a', 'b', 'c'].slice(-2)
< ["b", "c"]
```